Library/Formula/iojs.rb

```diff
@@ -0,0 +1,134 @@
+# Note that x.even are stable releases, x.odd are devel releases
```

Comment:
This line can be chopped. ` iojs ` appear to have gone for more sensible versioning.